qsdsan/sanunits/_clarifier.py

```diff
@@ -1040,12 +1040,13 @@ def _f_i(self):
         fx = xcod/self._mixed.COD
         corr = self._corr
         HRT = self._HRT
-        n_COD = corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(HRT*24*60))
+        # n_COD should not be negative
+        n_COD = max(0, corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(HRT*24*60)))
         f_i = 1 - (n_COD/100)
         return f_i
    
     def _run(self):
-        uf, of = self.outs
+        uf, of = self.outs # underflow, overflow
```

Comment:
@RaiSaumitra is it IWA's convention that the 0th effluent is underflow then the first is overflow? We usually (inherited from biosteam) rank flows from the lightest to heaviest, so overflow (less solids) then underflow (more solids)... but we can discuss/don't need to change it now